### ISSUE TITLE
version 0.23.0 appears to be broken

### ISSUE THREAD

**Issue by timur-khadimullin** [OctopusDeployLabs/terraform-provider-octopusdeploy#765]
**Describe the bug**
version 0.23.0 seems to be unavailable for `terraform init` to download

**Steps to reproduce**
1. create TF project, reference a version 
```hcl
terraform {
  required_version = ">= 1.6.0"

  required_providers {
    octopusdeploy = {
      source  = "OctopusDeployLabs/octopusdeploy"
      version = ">= 0.22.0"
    }
  }
}
```
3. run `terraform init` 

**Expected behavior**
`terraform init` completes successfully

**Logs and other supporting information**
```
Initializing modules...
........

Initializing provider plugins...
- Finding hashicorp/time versions matching ">= 0.12.0"...
- Finding octopusdeploylabs/octopusdeploy versions matching ">= 0.22.0"...
- Finding hashicorp/azurerm versions matching "~> 3.0"...
........
╷
│ Error: Failed to install provider
│ 
│ Error while installing octopusdeploylabs/octopusdeploy v0.23.0: could not
│ query provider registry for
│ registry.terraform.io/octopusdeploylabs/octopusdeploy: failed to retrieve
│ authentication checksums for provider: 404 Not Found returned from
│ github.com
╵
```

**Environment and versions:**
- OS: [`Linux`]
- Octopus Server Version: [N/A]
- Terraform Version: [e.g. `1.6.0`]
- Octopus Terraform Provider Version: [`0.23.0`]

**Additional context**
I don't see a 0.23.0 release on GitHub, so I wonder if this is related

**Comment by benPearce1** [comment]
Hi @timur-khadimullin, the release had to be removed. Unfortunately, there isn't an easy way to remove a version from the Hashicorp registry listing.
The current production release is `0.22.1`.

**Comment by benPearce1** [comment]
@timur-khadimullin I have published a new version tagged `0.23.1`, this should stop you getting a 404 with your current provider config.
